glossary.yml

```diff
@@ -708,8 +708,10 @@
   en:
     term: "backward-compatible"
     def: >
-      Software which is able to be used the same way as earlier versions of itself
-      without problems.
+      A property of a system, hardware or software, that allows for interoperability
+      with an older legacy system, or with input designed for such a system. 
+      For example a function written in python that can be run successfully with
```

Comment:
This should specify 'a function written in Python 3'. If it was written in Python 2, but also worked in Python 3, that would actually be forward-compatible.

Reply:
Good point! I will amend it, thanks.